src/plugins/intel_gpu/include/intel_gpu/graph/program.hpp

```diff
@@ -149,6 +150,7 @@ struct program {
     nodes_ordering& get_processing_order();
     uint32_t get_prog_id() { return prog_id; }
     stream& get_stream() { return *_stream; }
+    const stream& get_stream() const { return *_stream; }
```

Comment:
By the way why program has stream? Wansnt it a common data structure for multiple streams...

Reply:
We need it for memory copies on program build stage and weights reordering at least.